server/controllers/taskController.js

```diff
@@ -0,0 +1,66 @@
+const { Task } = require('../models')
+
+class TaskController {
+    static addTask(req, res, next) {
+        let input = {
+            title: req.body.title, 
+            category: req.body.category, 
+            UserId: req.UserId
+        }
+        Task.create(input)
+            .then(data => {
+                res.status(201).json(data)
+            })
+            .catch(next)
+    }
+
+    static getAllTasks(req, res, next) {
+        Task.findAll()
+            .then(data => {
+                res.status(200).json(data)
+            })
+            .catch(next)
+    }
+
+    static updateTask(req, res, next) {
+        let id = req.params.id
+        let input = {
+            title: req.body.title, 
+            category: req.body.category
+        }
+        Task.findOne({
```

Comment:
ga perlu findOne lagi yah, soalnya kan udah dihandle di middleware kan 
if task not found 😄